android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestSuggestionProvider.kt

```diff
@@ -142,6 +144,7 @@ internal data class SuggestionDetails(
     val fullKeyword: String,
     val isSponsored: Boolean,
     val icon: List<UByte>?,
+    val score: Float?,
```

Comment:
Hmm, `SuggestionDetails` is an internal class, and we don't use `details.score` anywhere—I don't think we need to add this just yet.

Reply:
kk